docker/Dockerfile

```diff
@@ -143,6 +144,11 @@ WORKDIR /project/avalon/common/python
 RUN echo "Building Avalon Common Python\n" \
   && make
 
+WORKDIR /project/avalon/common/crypto
+
+RUN echo "Building Avalon Common Crypto Python\n" \
+  && make
```

Comment:
@Ram-srini @rranjan3 . After merging this PR, will require related changes in block chain connector module to remove crypto dependencies. Can be done as separate PR